scalafmt-core/shared/src/main/scala/org/scalafmt/rewrite/RedundantBraces.scala

```diff
@@ -32,9 +32,16 @@ case object RedundantBraces extends Rewrite {
     def isIdentifierAtStart(value: String) =
       value.nonEmpty && (Character.isLetterOrDigit(value.head) || value.head == '_')
 
+    // hack for literal identifiers detection (`type`, `val`, ...)
+    def isLiteralIdentifier(arg: Term.Name): Boolean =
```

Comment:
Is there better solution for it?

Reply:
How about this?

```scala
arg.syntax.startsWith("`") && arg.syntax.endsWith("`")
```

Reply:
Oh, I didn't know about `Syntax` typeclass. Thank you very much!